lib/assert.js

```diff
@@ -807,11 +880,23 @@ function strict(...args) {
   innerOk(strict, args.length, ...args);
 }
 
+ArrayPrototypeForEach([
+  'ok', 'fail', 'equal', 'notEqual', 'deepEqual', 'notDeepEqual',
```

Comment:
Thinking more about this, we should probably keep `ok` as a self-reference property for now, although I doubt there are many folks relying on it, it doesn't cost us much to address it in a follow-up
```suggestion
// TODO(aduh95): take `ok` from `Assert.prototype` instead of a self-ref in the next major
assert.ok = assert;
ArrayPrototypeForEach([
  'fail', 'equal', 'notEqual', 'deepEqual', 'notDeepEqual',
```